### PR TITLE
Fixes #37706 - Add a template that enables download and run of a script

### PR DESCRIPTION
**Pull request by ShimShtein**
The main idea is to execute scripts from the Foreman server, similar to https://github.com/theforeman/foreman/pull/10208.

### CODE REVIEW

File: app/views/templates/script/run_downloaded_script.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+description_format: "Download the script from %{url} and run it locally"
+provider_type: script
+template_inputs:
+- name: url
+  description: URL of the script to download and execute
+  input_type: user
+  required: true
+%>
+
+set -o pipefail && curl -sS <%= input("url") %> | bash
```
Comment (ekohl) [inline app/views/templates/script/run_downloaded_script.erb:15]: Just to be sure, I'd quote the URL because it may contain `&`
```suggestion
set -o pipefail && curl -sS '<%= input("url") %>' | bash
```

File: .gitignore [hunk 2]
```diff
@@ -15,3 +15,4 @@ Gemfile.lock
 node_modules/
 package-lock.json
 coverage/
+public/
```
Comment (ekohl) [inline .gitignore:18]: This looks unrelated

**Review by ekohl — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (ShimShtein) [reply .gitignore]: It's still needed, so split it to a separate PR.

**Comment by ShimShtein** [comment]
Added quotes.

**Comment by ShimShtein** [comment]
@ekohl What do you think, would it be OK, or do we want to be more specific and just reuse the snippet from Foreman?

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**, works as expected

**Comment by ofedoren** [comment]
@ShimShtein, could you please create a RM ticket and attach it to the PR?

**Comment by ShimShtein** [comment]
@ofedoren Done.

**Review by ofedoren — APPROVED** [review]
Thanks, all, merging.
